### PR TITLE
Remove activities, add event destroy, change navbar, project events and project list design

### PR DESCRIPTION
**Pull request by dmytsuu**

### CODE REVIEW

File: src/app/components/event/event.ts [hunk 1]
```diff
@@ -24,74 +24,46 @@ export class Event implements OnInit {
...
-              private projectUserAPI: ProjectUserAPI,
               private eventService: EventService,
               private notifyService: NotificationService,
               private eventAPI: EventAPI,
               private router: ActivatedRoute,
               private redirect: Router) { }
 
   ngOnInit() {
+    this.router.parent.params.subscribe(params => {
+      if (params.id) {
```
Comment (Alexey-Lukin) [inline src/app/components/event/event.ts]: one line

File: src/app/components/project-events/project-events.scss [hunk 2]
```diff
@@ -2,16 +2,10 @@
 
 :host {
   .row {
-    height: calc(100vh - 11rem);
+    height: calc(100vh - 4.5rem);
   }
   .col {
```
Comment (Alexey-Lukin) [inline src/app/components/project-events/project-events.scss]:
```suggestion
  .col { max-width: 32% }
```

File: src/app/components/shared/event-list/event-list.ts [hunk 3]
```diff
@@ -76,16 +78,23 @@ export class EventList implements OnInit {
     this.events.splice(oldIndex, 0, event);
   }
 
+  destroyEvent = (event) => {
+    this.isDisabled = true;
+    if (!this.isProjectEvent(event)) { return this.isDisabled = false; }
```
Comment (Alexey-Lukin) [inline src/app/components/shared/event-list/event-list.ts]:
```suggestion
    if (this.isProjectEvent(event)) { this.prepareEventList(event) }
    this.isDisabled = false;
```

File: src/app/components/shared/event-list/event-list.html [hunk 4]
```diff
@@ -10,72 +10,81 @@
...
+              <li *ngIf="event.user_id" role="menuitem" class="dropdown-item">
+                <a (click)="updateEvent(event.id, {user_id: null}, null);">Unassign</a>
+              </li>
+              <div *ngIf="event.user_id" class="dropdown-divider"></div>
+              <li *ngFor="let item of projectUserService.projectUsers"
+                  (click)="updateEvent(event.id, {user_id: item.user_id}, null)"
+                  role="menuitem"
+                  class="dropdown-item"
+                  [class.bg-primary]="event.id === item.user_id">
+                <img [src]="item?.user?.image ? item.user.image : 'assets/bugno-logo.svg'" class="avatar rounded-circle">
```
Comment (viktor-shmigol) [inline src/app/components/shared/event-list/event-list.html]: [src]="item?.user?.image || 'assets/bugno-logo.svg'"

**Pull request merged**